frontend/mr-admin-flate/src/api/utkast/useUtkast.ts

```diff
@@ -1,8 +1,13 @@
 import { useQuery } from "@tanstack/react-query";
+import { UtkastDto } from "mulighetsrommet-api-client";
+import { ZodSchema } from "zod";
 import { QueryKeys } from "../QueryKeys";
 import { mulighetsrommetClient } from "../clients";
 
-export function useUtkast(utkastId?: string) {
+export function useUtkast(
+  schema: ZodSchema,
+  utkastId?: string,
+): { isLoading: boolean; data?: UtkastDto } {
```

Comment:
Kan undersøke om vi kan gjøre noe type-triksing for å få returtypen til å matche ZodSchema